lib/device_api/android/adb.rb

```diff
@@ -42,7 +42,7 @@ def self.getprop(serial)
 
         raise ADBCommandError.new(result.stderr) if result.exit != 0
 
-        lines = result.stdout.encode('UTF-8', invalid: :replace).split("\n")
+        lines = result.stdout.encode('UTF-16', 'UTF-8', invalid: :replace, replace: '').encode('UTF-8', 'UTF-16').split("\n")
```

Comment:
Line is too long. [125/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.